content/publication_resources/bibtex/lahti.bib

```diff
@@ -1,3 +1,20 @@
+@Article{König2024,
+  author =	 {Emilia König and Paulina Heponiemi and Sanni Kivinen and 
+                Jaakko Räkköläinen and Shea Beasley and Tuomas Borman and
+                Maria Carmen Collado and Vilja Hukkinen and Jouni Junnila and 
+                Leo Lahti and Marianna Norring and Virpi Piirainen and 
+                Seppo Salminen and Mari Heinonen and Anna Valros},
+  title =	 {Fewer culturable Lactobacillaceae species identified in faecal 
+                samples of pigs performing manipulative behaviour},
+  journal =	 {Scientific Reports},
```

Comment:
Put the {Lactobacillaceae} and {{Scientific Reports}} to maintain the uppercases